packages/quantic/force-app/main/default/lwc/quanticRefineModalContent/templates/disabledDynamicNavigation.html

```diff
@@ -6,18 +6,29 @@
   <template lwc:else>
     <template if:false={hideSort}>
       <div class="slds-var-m-bottom_small slds-clearfix">
-        <c-quantic-sort
-          class="slds-float_left"
-          engine-id={engineId}
-        ></c-quantic-sort>
+        <template lwc:if={shouldDisplaySort}>
```

Comment:
that was the logic followed for facets in the modal. If we want them there all the time I can change `this.shouldDisplaySort` to `this.hasCustomSortOptions` and then in the sortOptions getter change to :

`    return this.hasCustomSortOptions ? this.sortData : this.defaultSortOptions;` and add the defaultSortOptions like we do in the quanticSort

Reply:
To avoid confusing topics this not related to facets.
and yes we have this check because we wanna display the sort only after reading the sort related data from the store so we know the right sort options to pass to the sort component.

Like you mentioned, updating the name of the property will make this better.